### ISSUE TITLE
Pinch to Gesture is not working for Content View, when adding it to a scroll view

### ISSUE THREAD

**Issue by Deepak1799** [dotnet/maui#5614]
### Description

When adding pinch gesture recogniser to child (content view) of a scroll view, the pinch gesture is not working for the content view.

### Steps to Reproduce

1. Create File -> New -> MaUI App.
2. Add a scroll view control.
3. Add a content view with a content (image or some View) to scroll view.
4. Add pinch gesture recognizer to the content view as shown in the example: https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/gestures/pinch#creating-a-pinchtozoom-container
```
public class PinchToZoomContainer : ContentView
{
  ...

  public PinchToZoomContainer ()
  {
    var pinchGesture = new PinchGestureRecognizer ();
    pinchGesture.PinchUpdated += OnPinchUpdated;
    GestureRecognizers.Add (pinchGesture);
  }

  void OnPinchUpdated (object sender, PinchGestureUpdatedEventArgs e)
  {
    ...
  }
}
```
5. Now add a break point in the method **OnPinchUpdated** perform the pinch to zoom. The break point will not hit anymore.

### Version with bug

Preview 14 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 9.0

### Did you find any workaround?

No

### Relevant log output

```shell
NA
```

**Comment by mkdave** [comment]
I also have this problem if I add a PanContainer. (Android Only).  When I remove the PanContainer the pinch and zoom works fine.

**Comment by JeffIPV** [comment]
Any updates on this? It doesn't have to be added to scroll view.
Pinch to zoom gesture doesn't work when inheriting content view. 

Also doesn't work in BlazorWebView. Works fine on iOS.

**Comment by melixor** [comment]
Do you have any news about that?

**Comment by flipper09112** [comment]
Any updates?

**Comment by UnusualJustin** [comment]
I see BlazorWebView being mentioned here. Is the ContentView pinch gesture related to BlazorWebView? I cannot find any other instances of trying to enable user pinch-to-zoom on BlazorWebView (on Android).

For the standard WebView on Android, it requires accessing the Android namespace to enable zooming, but I don't see anything similar for BlazorWebView. 

If this is truly related to the original ContentView report, so be it, but if not, I believe a second issue should be opened to track the inability to zoom on Android.

**Comment by NandoLopes** [comment]
This bug is here for one whole year, and still nothing?

**Comment by Zhanglirong-Winnie** [comment]
Verified this issue with Visual Studio Enterprise 17.9.0 Preview 1.1. Not repro on android platform.

**Comment by rjhind** [comment]
What version of Maui is this fixed in?  The released 8.0.3?
